server/test/repository/ProgramRepositoryTest.java

```diff
@@ -202,7 +202,7 @@ public void getApplicationsForAllProgramVersions_withDateRange() {
     PaginationResult<Application> paginationResult =
         repo.getApplicationsForAllProgramVersions(
             program.id,
-            F.Either.Right(new PaginationSpec(10, 1)),
+            F.Either.Right(new PageNumberBasedPaginationSpec(10, 1)),
```

Comment:
inline doc the params please since there's 2 and it's not readable how they relate.

Probably not necessary for the identifier one since it can read pretty easy with 1 value.

Reply:
done